p2p/client.go

```diff
@@ -343,32 +347,40 @@ func (s *peer) GetSignaturesForDocument(ctx context.Context, model documents.Doc
 
 func (s *peer) validateSignatureResp(
 	model documents.Document,
-	receiver identity.DID,
+	receiver *types.AccountID,
 	header *p2ppb.Header,
-	resp *p2ppb.SignatureResponse) error {
-
+	resp *p2ppb.SignatureResponse,
+) error {
 	compatible := version.CheckVersion(header.NodeVersion)
 	if !compatible {
 		return version.IncompatibleVersionError(header.NodeVersion)
 	}
 
-	tm, err := model.Timestamp()
```

Comment:
I think we should keep this timestamp check. The way to check this with substrate is by:
- Getting the block from the number of the revoked key (if any)
- Get an extrinsic called "timestamp.set" that contains the unix epoch timestamp 
- Compare the model.Timestamp() with the one you just found

Here is an example at the bottom: https://centrifuge.subscan.io/block/1385880

Reply:
I would have the same questions that I have for the documents validator here (:

Reply:
Thanks!